raiden/transfer/node.py

```diff
@@ -564,11 +560,9 @@ def handle_contract_receive_channel_closed(
 
 
 def handle_receive_delivered(
-    chain_state: ChainState, state_change: ReceiveDelivered
+    chain_state: ChainState, state_change: ReceiveDelivered  # pylint: disable=unused-argument
 ) -> TransitionResult[ChainState]:
     """ Check if the "Delivered" message exists in the global queue and delete if found."""
-    queueid = QueueIdentifier(state_change.sender, CANONICAL_IDENTIFIER_UNORDERED_QUEUE)
-    inplace_delete_message_queue(chain_state, state_change, queueid)
```

Comment:
This basically means that we can entirely remove `Delivered` IMO. Need to verify but i think it's possible.

@hackaugusto thoughts?